### DISCUSSION TITLE
Newbie here... have 2 questions

### DISCUSSION THREAD

**Discussion by mefuzar** [coqui-ai/TTS#3645] [General]
Hello, nice to meet all of you.

I wanted to ask 2 questions, I am completely new to coqui AI TTS:
01. I run the server using Docker, I can choose a model and a speaker (have a long list of p### names).
I used VITS model and choose one of the names - does everything that I generate is free for commercial usage? Am I allowed to use the bundled voices with VITS? What kind of license does it require? these are voices that someone trained - can I use them?

02. I understood that XTTS v2 requires a special license but for free - is VITS my best choice? are the bundled voices are the best choice? 

In other words: I I would like to get the best output possible (but under free license constraint). What should I do?

Thank you very much in advance 🙏🙏🙏
